core/templates/pages/exploration-editor-page/editor-tab/services/responses.service.ts

```diff
@@ -392,70 +392,6 @@ angular.module('oppia').factory('ResponsesService', [
             }, callback);
           });
         }
-
-        // If the interaction is DragAndDropSortInput, update the answer groups
-        // to refer to the new answer options.
-        if (StateInteractionIdService.savedMemento === 'DragAndDropSortInput' &&
```

Comment:
Why exactly is this removed?

Reply:
Seems like no special case handling is required for DragAndDropSortInput interactions. The answer choices seem to update correctly. Also, the values set as part of this special case handling were invalid (see empty list being set L448 and empty strings being set in L438 and L443).

Reply:
I'm still trying to find out why this was added in the first place.

Reply:
Update: Figured out why this is required. It is to handle updates to customization args. I'll need to add this back and make some modifications similar to that of ItemSelectionInput interaction (see code above) to handle that scenario.

Reply:
@aks681 I've added it back and made few changes. PTAL!